### DISCUSSION TITLE
Release v2.7.1

### DISCUSSION THREAD

**Discussion by mika-f** [mika-f/blender-drag-and-drop#46] [Announcements]
## CHANGES

* Support to Blender 3.6.0 (#42 )

## NOTES

* Restart Blender after updating add-on. Due to limitations of Windows, changes will not take effect until restart.

<hr /><em>This discussion was created from the release <a href='https://github.com/mika-f/blender-drag-and-drop/releases/tag/v2.7.1'>Release v2.7.1</a>.</em>

**Comment by kenhes** [comment]
Nothing happens after installing it. Not in the installed list. (I've restarted Blender.)

**Reply by mika-f** [reply]
What version of Blender is used?

**Comment by kenhes** [comment]
I didn't realize I was auto updated. I'm on 3.6.1 Release Candidate.

**Comment by mika-f** [comment]
It happens :)
